frontend/control-center/src/translations.ts

```diff
@@ -0,0 +1,887 @@
+      instagramConfigurationText3:
+        'Une intégration de webhook afin que la plateforme puisse ingérer les messages de vos pages Instagram.',
+      instagramConfigurationText4: 'Un jeton de page pour chaque page Facebook que vous souhaitez intégrer.',
+      instagramConfigurationText5: `Vérifiez la documentation d'Airy`,
+      instagramConfigurationText6: `pour plus d'informations.`,
+
+      //Twilio
+
+      twilioPhoneNumber: 'Numéro de téléphone de Twilio',
+      twilioPhoneNumberPlaceholder: 'Numéro acheté +123456789',
+      connectSmsNumber: 'Connecter le numéro Sms',
+      updateSmsNumber: 'Mise à jour du numéro Sms',
+
+      connectWhatsapp: 'Connecter Whatsapp',
+      connectWithTwilio: 'Connectez-vous avec Twilio First',
+      twilioConfigurationText: 'Avant de connecter un numéro pour les SMS ou Whatsapp, vous devez ajouter un',
+      twilioConfigurationText2: `Token d'authentification Twilio`,
+      twilioConfigurationText3: 'au',
+      twilioConfigurationText4: 'airy.yaml',
+      twilioConfigurationText5: 'fichier.',
+      twilioConfigurationText6: 'Après cela, vous devez acheter un numéro.',
+      twilioConfigurationText7: 'Vérifiez',
+      twilioConfigurationText8: `Vérifiez la documentation d'Airy`,
+      twilioConfigurationText9: `pour plus d'informations.`,
+      connectWhatsappNumber: 'Connecter le numéro Whatsapp',
+      updateWhatsappNumber: 'Mise à jour du numéro Whatsapp',
+
+      //Connectors
+
+      noResults: 'Résultat non trouvé.',
+      noResultsTerm: 'Essayez de rechercher un autre terme.',
+      noConnectorsFound: 'Aucun connecteur trouvé',
+      noConnectorsFoundTerm: `Vous n'avez pas de connecteurs installés, veuillez ouvrir l'application`,
+      noConnectorsFoundMore: 'et explorer davantage.',
+      installed: 'Installé',
+      notInstalled: 'Non installé',
+
+      //NotFound
+
+      notFound: `Oups! Nous n'avons pas pu trouver ça ici.`,
+
+      //Status
+
+      status: 'Statut',
+      componentName: 'Nom du composant',
+      healthStatus: 'État de santé',
+      enabled: 'Activé',
+
+      //Webhooks
+
+      errorOccurred: `Une erreur s'est produite`,
+      successfullySubscribed: 'Souscription réussie!',
+      subscribeWebhook: 'Abonnement au Webhook',
+      updateWebhook: 'Mise à jour du Webhook',
+      unsubscribeWebhook: 'Désabonnement Webhook',
+      unsubscribeWebhookText: 'Vous êtes sûr de',
+      unsubscribeWebhookText2: 'vouloir vous désabonner ',
+      unableToUnsubscribeWebhook: 'Impossible de désabonner le Webhook',
+      subscribeCapital: `S'abonner à`,
+      updateCapital: 'Mise à jour',
+      subscribing: `S'abonner...`,
+      updating: 'Mise à jour...',
+      webhookCapslock: 'WEBHOOK',
+      allEvents: 'TOUS LES ÉVÉNEMENTS',
+
+      //Inbox
+
+      //Tags
+      deleteTag: 'Êtes-vous sûr de vouloir supprimer définitivement ce tag?',
```

Comment:
deleteTag: 'Êtes-vous sûr de vouloir supprimer définitivement ce tag ?',

_In French, exclamation points, question marks, colons, and semi-colons (all forms of "high" punctuation) should always have a space preceding them_